android/ReactAndroid/build.gradle

```diff
@@ -306,7 +306,11 @@ dependencies {
     api "com.squareup.okhttp3:okhttp:${OKHTTP_VERSION}"
     api "com.squareup.okhttp3:okhttp-urlconnection:${OKHTTP_VERSION}"
     api 'com.squareup.okio:okio:1.14.0'
-    api 'org.webkit:android-jsc:r174650'
+    compile 'org.webkit:android-jsc:r174650'
+    compile 'expolib_v1.com.squareup.okhttp3:okhttp:3.6.0'
```

Comment:
Can these expolib imports use `api` instead of `compile`? The `compile` option is going away so if we don't need to use it, that's better.